operator/pkg/gateway-api/secret.go

```diff
@@ -92,3 +101,51 @@ func (r *secretSyncer) enqueueRequestForGatewayTLS() handler.EventHandler {
 		return reqs
 	})
 }
+
+func enqueueOwningSecretFromLabels() handler.EventHandler {
+	return handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, o client.Object) []reconcile.Request {
+		requests := []reconcile.Request{}
+		labels := o.GetLabels()
+
+		if labels != nil {
```

Comment:
If i read this correct labels not being nil is the success path? Would it be a readibility improvement to do
```
if labels == nil {
  return
}
```

So the success path stays left aligned (i almost overread it)

Reply:
Thanks. fully agree. while on it, i also return if the namespace & name labels aren't present. This way the success path is fully left-aligned.